### ISSUE TITLE
Support for OpenID Connect (OIDC) / OAuth2

### ISSUE THREAD

**Issue by alexanderadam** [bigcapitalhq/bigcapital#237]
Does BigCapital have support for OIDC or is it planned to add support for it?

**Comment by poudelprakash** [comment]
@abouolia what is core team's take on this?
I would like to work on this if core team is open to adding this feature..

**Comment by abouolia** [comment]
@poudelprakash Yes, we want to do it but now we have other priorities we should work on them, I would love to review your PR, if needed any help or questions please let me know.
